app/routers/list.py

```diff
@@ -0,0 +1,75 @@
+from fastapi import APIRouter, Depends, HTTPException, Response, status
+from sqlalchemy.orm import Session
+
+from app.database import get_db
+from app.models import List
+from app.schemas import List as ListSchema
+from app.schemas import ListCreate, ListWithoutTodos
+
+router = APIRouter(prefix="/lists", tags=["Lists"])
+
+@router.post("/", status_code=status.HTTP_201_CREATED)
+def create_list(list: ListCreate, db: Session = Depends(get_db)) -> ListWithoutTodos:
+    new_list = List(**list.model_dump())
```

Comment:
instead of `List(...)` you can use `List.model_validate(...)` which will then already complain about missing title and so on, depending on what you have defined in your pydantic schema.

Reply:
Like that you dont need the `if new_list.title...` block below